installing/installing_rhv/installing-rhv.adoc

```diff
@@ -0,0 +1,13 @@
+[id="installing-rhv"]
+= Installing a cluster on Red Hat Virtualization
+include::modules/common-attributes.adoc[]
+:context: installing-rhv
+
+toc::[]
+
+In {product-title} version {product-version}, you can install a cluster
+in a Red Hat Virtualization (RHV) environment.
+
+The installer that deploys OCP on RHV automates the process using what’s called “installer-provisioned infrastructure.” This installer is available for Linux and macOS only.
```

Comment:
s/OCP/{product-title}/

I'm not sure I'd do

> using what’s called “installer-provisioned infrastructure.”

maybe

_automates the process by using installer-provisioned infrastructure._